tests/unit/core/test_deploy_export.py

```diff
@@ -349,4 +464,47 @@ def test_hf_bert(self, tmpdir, df_type):
         # Generate filename in the temporary directory.
         tmp_file_name = str(tmpdir.mkdir("export").join("bert"))
         # Test export.
-        self.__test_export_route(module=bert, out_name=tmp_file_name, mode=df_type, input_example=input_example)
+        self.__test_export_route(
+            module=bert, out_name=tmp_file_name, mode=df_type, input_example=input_example, onnx_opset=11
+        )
+
+    @pytest.mark.unit
+    @pytest.mark.run_only_on('GPU')
+    @pytest.mark.parametrize(
+        "df_type", [DF.ONNX, DF.TORCHSCRIPT, DF.PYTORCH]  # , pytest.param(DF.TRTONNX, marks=requires_trt)]
+    )
+    def test_waveglow(self, tmpdir, df_type):
+        url = "https://api.ngc.nvidia.com/v2/models/nvidia/waveglow_ljs_256channels/versions/2/files/waveglow_256channels_ljs_v2.pt"
+        ptfile = "./waveglow_256channels_ljs_v2.pt"
+        if not Path(ptfile).is_file():
+            urllib.request.urlretrieve(url, ptfile)
+
+        waveglow = load_and_setup_model(ptfile)
+
+        # 80 mel channels, 620 mel spectrograms ~ 7 seconds of speech
+        mel = torch.randn(1, 80, 620).cuda()
+        stride = 256  # value from waveglow upsample
+        n_group = 8
+        z_size2 = (mel.size(2) * stride) // n_group
+        z = torch.randn(1, n_group, z_size2, 1).cuda()
+        sigma_infer = 1.0
+
+        with torch.no_grad():
+            # run inference to force calculation of inverses
+            waveglow.infer(mel, sigma=sigma_infer)
+
+            # export to ONNX
+            convert_1d_to_2d_(waveglow)
+            fType = types.MethodType
+            waveglow.forward = fType(infer_onnx, waveglow)
```

Comment:
why do we need to call it like this? Doesn't this change waveglow.forward method to infer_onnx?